### ISSUE TITLE
Allow Specific Namespace tags to be overwritten with new values

### ISSUE THREAD

**Issue by official-elinas** [hydrusnetwork/hydrus#1306]
I modified the Gelbooru parser to grab the score tag and added it to a namespace `score:<num>` 

Realized this would create duplicate entries based on how tagging works. The reason I would like to collect score values is for a metric of "quality" for training datasets in Stable Diffusion/as a general metric.

Effectively, this would be an option to select a namespace tag and allow it to be overwritten by new values when imported.

**Comment by official-elinas** [comment]
Correct. All I did was modify the parser to get the score value and create it as a namespaced tag in my own "score" tag repo. Ideally, when I parse an existing image, instead of adding a second, or even third...etc `score:123` tag, it would update it. 

I can share it on discord or here, I guess. I'm Elinas#5898 on hydrus discord. Might be easier there to chat. Also, thanks for replying to this issue, never thought it would get any traction.

**Comment by Chad90b** [comment]
> @official-elinas you should share things here so that everyone can see. To be honest, your comment left me confused, so I need to see it in practice.

i believe what he wants is a system that can replace an existing value rather than adding a new second value, the second time you parse the same post.

if you parsed one post five times, you would today end up with five scores on the same file:
`score:123`
`score:229`
`score:499`
`score:800`
`score:1002`

i understand that he wants the older ("outdated") scores to automatically be removed/overwritten whenever a post is re-parsed **so he only keeps the latest score value and all the other values are removed** in a single entry. this might not be an ideal fit for the tagging system, but in either case you would need a code overhaul to support this specific request

**Comment by official-elinas** [comment]
I know you can have multiple services but that's not a good use for it. Rating cannot replicate score as it's limited to 20 "stars"
